cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -39,6 +49,8 @@
 import java.util.function.Supplier;
 import javax.annotation.Nullable;
 
+import static java.nio.file.LinkOption.NOFOLLOW_LINKS;
```

Comment:
I think we don't use static imports, was it not caught by checkstyle?

Reply:
There is no checkstyle rule for that in checksyle.xml. I've removed the static import now, since it was no longer needed.